osu.Game/Configuration/GameConfigManager.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Configuration;
+using osu.Framework.Configuration.Tracking;
+using osu.Framework.Extensions;
+using osu.Framework.Platform;
+using osu.Game.Overlays;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Screens.Select;
+
+namespace osu.Game.Configuration
+{
+    public class GameConfigManager : IniConfigManager<GameSetting>
+    {
+        protected override void InitialiseDefaults()
+        {
+            // UI/selection defaults
+            Set(GameSetting.Ruleset, 0, 0, int.MaxValue);
+            Set(GameSetting.Skin, 0, 0, int.MaxValue);
+
+            Set(GameSetting.BeatmapDetailTab, BeatmapDetailTab.Details);
+
+            Set(GameSetting.ShowConvertedBeatmaps, true);
+            Set(GameSetting.DisplayStarsMinimum, 0.0, 0, 10, 0.1);
+            Set(GameSetting.DisplayStarsMaximum, 10.0, 0, 10, 0.1);
+
+            Set(GameSetting.RandomSelectAlgorithm, RandomSelectAlgorithm.RandomPermutation);
+
+            Set(GameSetting.ChatDisplayHeight, ChatOverlay.DEFAULT_HEIGHT, 0.2, 1);
+
+            // Online settings
+            Set(GameSetting.Username, string.Empty);
+            Set(GameSetting.Token, string.Empty);
+
+            Set(GameSetting.SavePassword, false).ValueChanged += val =>
+            {
+                if (val) Set(GameSetting.SaveUsername, true);
+            };
+
+            Set(GameSetting.SaveUsername, true).ValueChanged += val =>
+            {
+                if (!val) Set(GameSetting.SavePassword, false);
+            };
+
+            Set(GameSetting.ExternalLinkWarning, true);
+
+            // Audio
+            Set(GameSetting.VolumeInactive, 0.25, 0, 1, 0.01);
+
+            Set(GameSetting.MenuVoice, true);
+            Set(GameSetting.MenuMusic, true);
+
+            Set(GameSetting.AudioOffset, 0, -500.0, 500.0, 1);
+
+            // Input
+            Set(GameSetting.MenuCursorSize, 1.0, 0.5f, 2, 0.01);
+            Set(GameSetting.GameplayCursorSize, 1.0, 0.5f, 2, 0.01);
+            Set(GameSetting.AutoCursorSize, false);
+
+            Set(GameSetting.MouseDisableButtons, false);
+            Set(GameSetting.MouseDisableWheel, false);
+
+            // Graphics
+            Set(GameSetting.ShowFpsDisplay, false);
+
+            Set(GameSetting.ShowStoryboard, true);
+            Set(GameSetting.BeatmapSkins, true);
+            Set(GameSetting.BeatmapHitsounds, true);
+
+            Set(GameSetting.CursorRotation, true);
+
+            Set(GameSetting.MenuParallax, true);
+
+            Set(GameSetting.SnakingInSliders, true);
+            Set(GameSetting.SnakingOutSliders, true);
```

Comment:
Wanna remove these now?

Reply:
:eyes: I screwed up the merge. Yes, will remove the zombies again.